### PR TITLE
Pin leptonica

### PR DESCRIPTION
**Pull request by CurtLH**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by CurtLH** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,6 +14,8 @@ source:
 build:
   number: 2
```
Comment (isuruf) [inline recipe/meta.yaml]: This needs to be bumped to 3

Comment (CurtLH) [reply recipe/meta.yaml]: Done.

Comment (isuruf) [reply recipe/meta.yaml]: Hmm, build number 3 is already there. Can you increase it to 4?

**Comment by CurtLH** [comment]
@isuruf -- I'll wait for you to give the go ahead to merge this one. :+1:

**Review by isuruf — APPROVED** [review]

**Pull request merged**